oshmem/runtime/runtime.h

```diff
@@ -237,6 +237,16 @@ OSHMEM_DECLSPEC int oshmem_shmem_register_params(void);
 
 #endif  /* OSHMEM_PARAM_CHECK */
 
+#define RUNTIME_SHMEM_NOT_IMPLEMENTED_API_ABORT()                    \
+    do {                                                             \
+        SHMEM_API_ERROR("Called non-implemented API: %s", __func__); \
+        oshmem_shmem_abort(OSHMEM_ERR_NOT_IMPLEMENTED);              \
+    } while (0)
+
+#define RUNTIME_SHMEM_NOT_IMPLEMENTED_API_ABORT_RET_SIZE_T() \
+    RUNTIME_SHMEM_NOT_IMPLEMENTED_API_ABORT();               \
+    return SIZE_MAX
+
```

Comment:
Suggest proper scoping using do-while (like above)

```suggestion
#define RUNTIME_SHMEM_NOT_IMPLEMENTED_API_ABORT_RET_SIZE_T() \
    do {  \
        RUNTIME_SHMEM_NOT_IMPLEMENTED_API_ABORT();     \
        return SIZE_MAX; \
    } while (0)

```